lib/nexpose/common.rb

```diff
@@ -25,20 +25,25 @@ module Scope
   # non-members.
   class Email
     # Send as file attachment or zipped file to individuals who are not members
-    # of the report access list. One of: file|zip
+    # of the report access list. One of: [String] 'file' | 'zip'
     attr_accessor :send_as
     # Send to all the authorized users of sites, groups, and assets.
+    # Ex. : Fixnum [1 | 0]
     attr_accessor :to_all_authorized
-    # Send to users on the report access list.
+    # Send to users on the report access list. Ex. : [String] 'file' | 'zip'
     attr_accessor :send_to_acl_as
-    # Format to send to users on the report access list. One of: file|zip|url
+    # Format to send to users on the report access list.
+    # One of: [String] 'file' | 'zip'
     attr_accessor :send_to_owner_as
 
     # Sender that e-mail will be attributed to.
+    # String ['abc.123@gmail.com']
     attr_accessor :sender
-    # SMTP relay server.
+    # SMTP relay server. This is necessary to avoid security issue for
+    # internal network.
+    # String ['abcd.server.com']
     attr_accessor :smtp_relay_server
-    # Array of report recipients (i.e., not already on the report access list).
+    # Array[String] of report recipients (i.e., not already on the report access list).
```

Comment:
Change this to:
```
[Array[String]]
```
for the right highlighting in generated documentation. Probably want to specify the recipients are expected as email addresses.